docs/sources/developers/plugins/_index.md

```diff
@@ -28,12 +29,12 @@ You can create several types of plugins, including:
 
 The following topics are covered in this guide:
 
-- **[Get started with plugins]({{< relref "./get-started-with-plugins" >}})** - Start developing Grafana plugins with the [Create-plugin](https://www.npmjs.com/package/@grafana/create-plugin) tool.
+- **[Get started with plugins]({{< relref "./get-started-with-plugins" >}})** - Start developing Grafana plugins with the [create-plugin](https://www.npmjs.com/package/@grafana/create-plugin) tool.
 - **[Introduction to plugin development]({{< relref "./introduction-to-plugin-development" >}})** - Learn the fundamentals of Grafana plugin development: frontend and backend development processes, data frames, error handling, and more.
```

Comment:
> Learn the fundamentals of Grafana plugin development: frontend and backend development processes

I have a problem that this just isn't true currently, theres nothing on frontend and I wouldn't really say its the development process.

Reply:
updated to "Learn the fundamentals of Grafana plugin development: backend development, data frames, error handling, and more."